Mathlib/Analysis/Fourier/FourierTransform.lean

```diff
@@ -167,7 +167,6 @@ theorem fourierIntegral_continuous [TopologicalSpace.FirstCountableTopology W] (
   apply continuous_of_dominated
   · exact fun w => ((fourier_integral_convergent_iff he hL w).mp hf).1
   · refine' fun w => ae_of_all _ fun v => _
-    · exact fun v => ‖f v‖
     · rw [norm_smul, Complex.norm_eq_abs, abs_coe_circle, one_mul]
```

Comment:
no more need for a bullet here.

Reply:
Hmm, it's not strictly necessary, but it does show clearly that `refine'` generated two subgoals, right? I'd consider that more readable (unless I'm misunderstanding). :)